parse.go

```diff
@@ -24,48 +22,71 @@ type Section interface {
 func Parse(v string) (GppContainer, error) {
 	var gpp GppContainer
 
-	gpp.SectionStrings = strings.Split(v, "~")
+	sectionStrings := strings.Split(v, "~")
 
-	buff := []byte(gpp.SectionStrings[0])
-	decoded := make([]byte, base64.RawURLEncoding.DecodedLen(len(buff)))
-	n, err := base64.RawURLEncoding.Decode(decoded, buff)
+	bs, err := util.NewBitStreamFromBase64(sectionStrings[0])
 	if err != nil {
 		return gpp, err
 	}
-	decoded = decoded[:n:n]
-
-	bs := util.NewBitStream(decoded)
+	if bs.Len() < 3 {
```

Comment:
Can the check here use a constant in place of the 3?

Reply:
sure